### PR TITLE
Fix tarball downloads

### PR DESCRIPTION
**Pull request by targos**
- fix: properly download package tarballs
- lookup: skip pug if >=16

### CODE REVIEW

**Comment by targos** [comment]
@nodejs/citgm

File: lib/grab-project.js [hunk 1]
```diff
@@ -24,18 +31,23 @@ export async function grabProject(context) {
       `${context.module.name} ${packageManager}:`,
       `Downloading project: ${packageName}`
     );
+
+    // Default timeout to 10 minutes if not provided.
+    const timeout = context.options.timeout || 1000 * 60 * 10;
+
+    if (packageName.startsWith('http:') || packageName.startsWith('https://')) {
```
Comment (ljharb) [inline lib/grab-project.js:38]: why a double slash on one but not the other?

Comment (targos) [reply lib/grab-project.js:38]: I don't know, a mistake. Which version would you suggest?

Comment (ljharb) [reply lib/grab-project.js:38]: not sure it matters; I’d probably do `/^https?:\/\//.test()` personally but i don’t have any concrete argument why that’s better. i was just curious about the inconsistency.

**Comment by targos** [comment]
@nodejs/citgm

**Review by richardlau — APPROVED** [review]

**Pull request merged**